.env.example

```
@@ -41,6 +41,11 @@ STRIPE_ACCOUNT=acct_
 STRIPE_PUBLISHABLE_KEY=pk_test_
 STRIPE_SECRET_KEY=sk_test_
 STRIPE_WEBHOOK_SECRET=whsec_
+# CSV of all webhook signing secrets (platform + Connect endpoints). When set,
+# takes precedence over STRIPE_WEBHOOK_SECRET.
+STRIPE_WEBHOOK_SECRETS=
+STRIPE_API_VERSION=2026-03-25.dahlia
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The STRIPE_API_VERSION key should go before the STRIPE_PUBLISHABLE_KEY key

(UnorderedKey)